hack/tools/release/weekly/main.go

```diff
@@ -169,9 +169,32 @@ func run() int {
 		merges[key] = append(merges[key], formatMerge(body, prNumber))
 	}
 
+	// fetch the current branch
+	out, err = exec.Command("git", "rev-parse", "--abbrev-ref", "HEAD").CombinedOutput()
+	if err != nil {
+		fmt.Println("Error")
+		fmt.Println(err)
+		fmt.Println(string(out))
+		return 1
+	}
+
+	var branch string
+	outLines = strings.Split(string(out), "\n")
+	for _, line := range outLines {
+		line = strings.TrimSpace(line)
+		if line != "" {
+			branch = line
+			break
+		}
+	}
```

Comment:
I'm a bit confused what this block of code is doing. Does it set the branch to the last line of the branch name? I think it may be better to include the entire branch name if it spans multiple lines.

Reply:
it was just removing any un-necessary lines if comes, but I think that is not required as this command will only return with current branch only. so without this it will also work fine.